sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-response/_tests_/ece-response-integration.scheduler.e2e-spec.ts

```diff
@@ -135,9 +160,130 @@ describe(
       expect(processResult).toStrictEqual([expectedResult]);
       // Expect the delete method to be called.
       expect(sftpClientMock.delete).toHaveBeenCalled();
+      // Expect the notifications to be created.
+      const notifications = await getUnsentECEResponseNotifications(db);
+      // TODO: When bulk send email is implemented, we must always expect 1 notification
+      // record to be created.
+      expect(notifications).toHaveLength(
+        locationCONF.integrationContacts.length,
+      );
+      const updatedDisbursement = await db.disbursementSchedule.findOne({
+        select: { coeStatus: true },
+        where: { id: disbursement.id },
+      });
+      // Expect the COE status of the updated disbursement to be completed.
+      expect(updatedDisbursement.coeStatus).toBe(COEStatus.completed);
+    });
+
+    it("Should process an ECE response file and confirm the enrolment and create notification when a disbursement has multiple detail records.", async () => {
+      // Arrange
+      // Enable integration for institution location
+      // used for test.
+      await enableIntegration(locationMULT, db);
+      const confirmEnrolmentResponseFile = path.join(
+        process.env.INSTITUTION_RESPONSE_FOLDER,
+        locationMULT.institutionCode,
+        ECE_RESPONSE_FILE_NAME,
+      );
+
+      // Create disbursement to confirm enrolment.
+      const application = await saveFakeApplicationDisbursements(
+        db.dataSource,
+        {
+          // Save disbursement with custom award value to validate tuition remittance.
```

Comment:
its not a custom value right

Reply:
Can you please be elaborate on why you consider this award value not being custom?

In my view it was replacing the default value of award ($1) provided by the factory to specifically test one more scenarios in my suite and hence I called it custom.

Reply:
IMO, I still don't see it as a custom.  I see it as an actual award.